### ISSUE TITLE
Licence query with regards to pklib.h

### ISSUE THREAD

**Issue by Kakolukia91** [ladislav-zezula/StormLib#284]
Hello.

I've got a question about licencing with regards to using functionality from this repo.

The specific functionality I'm interested in is PKWARE DCL (from `pklib.h`). From what I can see/read this was provided by PKWARE software company (although I can't find any resources on DCL licence from the company's website).

As this repo licence is MIT, does that mean using the functionality from `pklib.h` falls under the same licence, or would that need a licence from PKWARE?

Thanks.

**Comment by ladislav-zezula** [comment]
I honestly have no idea. I obtained the sources by reverse engineering 25 years ago. Since then, no one from PKWARE ever contacted me about it. The library existed in the age of the DOS operating system.

TLDR: Don't bother about license :-)

**Comment by Kakolukia91** [comment]
Thanks for the reply @ladislav-zezula 

I've got specific use case that requires me to use PKWARE, so wanted to verify the licence requirements.

I'll give the one in this repo a try.

Thank you.
